### DISCUSSION TITLE
Mask is based on grad-cam

### DISCUSSION THREAD

**Discussion by snassimr** [jacobgil/pytorch-grad-cam#314] [Q&A] [answered]
Hi everyone ,

Surprisingly I struggle to create mask based on grad-cam output . Any code snippet on this ?

![image](https://user-images.githubusercontent.com/6830626/185985955-e036da31-aa55-4d0d-b936-7778b7e00051.png)

**Comment by CristianCosci** [comment]
Hi, i'm doing exactly this. 
Tomorrow I'll share it with you.

**Comment by CristianCosci** [comment, marked as answer]
Hi!
As i said yesterday i prepare a .ipynb for this task.
Check my fork of this repo at https://github.com/CristianCosci/pytorch-grad-cam.git and go to `personal_files/` folder and see the `grad_cam_operations.ipynb` notebook.
I created it to use as an example of applications of a filter only in the area returned by a grad_cam as mask. I think it is exactly what you need.

For every other question please ask me, and don't forget to star my repo! Thanks.

**Comment by snassimr** [comment]
Thanks , Christian . Great work !!! I have a question . What the usage of filter in "Create an example of a filter to see the applications of created mask" section . Why using original image isn't enough as it happens in "Isolating the img area using mask" section. I wanted to put the question on your repo , but I don't see "Discussion" tab there.

**Reply by CristianCosci** [reply]
Obviously no problem I will answer to you.
I have enabled the "discussion" tab in my repository, so post your question and I provide the answer there (so as not to mess up this discussion tab).
